src/pages/Trade/Trade.tsx

```diff
@@ -275,6 +275,13 @@ function Trade() {
                         showResizeable={!isCandleDataNull}
                         enable={{
                             bottom: !isChartFullScreen,
+                            top: false,
+                            left: false,
+                            topLeft: false,
+                            bottomLeft: false,
+                            right: false,
+                            topRight: false,
+                            bottomRight: false,
```

Comment:
this is the actual fix (to disable all other pivot points. such that only the needed pivot points are rendered and not all.

<img width="993" alt="image" src="https://github.com/CrocSwap/ambient-ts-app/assets/45405267/c63f4dc0-db98-4554-b546-47bd679b1550">